docs/devops/snippets/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Set the base image as the .NET 5.0 SDK (this includes the runtime)
+FROM mcr.microsoft.com/dotnet/sdk:5.0 as build-env
+
+# Copy everything and publish the release (publish implicitly restores and builds)
+COPY . ./
+RUN dotnet publish ./DotNet.GitHubAction/DotNet.GitHubAction.csproj -c Release -o out --no-self-contained
+
+# Label the container
+LABEL maintainer="David Pine <david.pine@microsoft.com>"
+LABEL repository="https://github.com/dotnet/samples"
+LABEL homepage="https://github.com/dotnet/samples"
+
+# Label as GitHub action
+LABEL com.github.actions.name="The name of your GitHub Action"
+# Limit to 160 characters
+LABEL com.github.actions.description="The description of your GitHub Action."
```

Comment:
Any information of where these would show or how they're different from the ones in `action.yml`?

Reply:
Yeah, they probably are old syntax - the *action.yml* is really what matters. But from a standalone aspect of the .NET container, having labels on it are nice too. It helps to identify the container, and certainly doesn't hurt to have them.